pkgs/tools/misc/screen/default.nix

```diff
@@ -36,14 +29,11 @@ stdenv.mkDerivation rec {
            -e 's/^as_fn_error .. \("cannot run test program while cross compiling\)/$as_echo \1/g'
     ''; # "
 
-  buildInputs = [ ncurses ] ++ lib.optional stdenv.isLinux pam
-                            ++ lib.optional stdenv.isDarwin utmp;
-
-  # Build fails due to missing dependencies on autogenerated header:
-  #   screen.h:48:10: fatal error: comm.h: No such file or directory
-  # It will be fixed in next screen-4.9.0 release. The patches are
-  # invasive. Disabling parallelism for 4.8.0.
-  enableParallelBuilding = false;
+  buildInputs = [
+    autoreconfHook
```

Comment:
I'm pretty sure that this belongs to `nativeBuildInputs`, correct?

Reply:
Oh!  Yes, of course!  Fixed.  Thanks!